sdk/storage/src/blob/blob/block_list.rs

```diff
@@ -73,22 +65,29 @@ where
 #[cfg(test)]
 mod test {
     use super::*;
+    use bytes::Bytes;
 
     #[test]
     fn to_xml() {
         let mut blocks = BlockList { blocks: Vec::new() };
         blocks
             .blocks
-            .push(BlobBlockType::Committed(Vec::from(b"numero1" as &[u8])));
+            .push(BlobBlockType::new_committed(Bytes::from_static(b"numero1")));
         blocks
             .blocks
-            .push(BlobBlockType::Uncommitted(Vec::from(b"numero2" as &[u8])));
+            .push(BlobBlockType::new_uncommitted(Bytes::from_static(
+                b"numero2" as &[u8],
```

Comment:
nit: I think the as &[u8] is not needed. You removed it in the block above.

Reply:
True! Fixed! 👍